src/styles/main.scss

```diff
@@ -1,3 +1,209 @@
+@import "stars";
+
+$buy--color: #00acdc;
+$text--color--one: #060b35;
+$text--color--two: #616070;
+$header--bcg--color: #fff;
+$nav__item--color: #000;
+$card__hover: #34568b;
+
 body {
   margin: 0;
+  font-size: 10px;
+  line-height: 14px;
+  color: $text--color--one;
+}
+
+html {
+  font-family: "Roboto", "Arial", sans-serif;
+}
+
+.header {
+  background-color: $header--bcg--color;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+  transition-duration: 300ms;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  line-height: 60px;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: $text--color--one;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+.logo__image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__item {
+  color: $nav__item--color;
+  font-size: 12px;
+  font-style: blod;
+  font-weight: 550;
+  line-height: normal;
+  text-transform: uppercase;
+  margin-left: 20px;
+  font-family: Roboto, sans-serif;
+}
+
+.nav__link.is-active {
+  color: $buy--color;
+}
+
+.nav__link.is-active:after {
+  content: "";
+  position: absolute;
+  background-color: $buy--color;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4px;
+  border-radius: 2px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link:hover {
+  color: $buy--color;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px;
+  display: block;
+}
+
+.card {
+  width: 200px;
+  height: 406px;
+  flex-shrink: 0;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  box-sizing: border-box;
+  transition-duration: 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__info {
+    color: $text--color--one;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 40px 16px 4px;
```

Comment:
you have margin-top for info and margin-bottom for image - there is margin collapse. Use one direction margins only to make it clear